content/docs/1.1.1/deploy/upgrade/_index.md

```diff
@@ -7,16 +7,25 @@ Here we cover how to upgrade to latest Longhorn from all previous releases.
 
 # Upgrading Longhorn
 
-There are normally two steps in the upgrade process: first upgrade Longhorn manager to the latest version, then upgrade the Longhorn engine to the latest version using the latest Longhorn manager.
+There are normally two steps in the upgrade process: first upgrade Longhorn manager to the latest version, then manually upgrade the Longhorn engine to the latest version using the latest Longhorn manager.
 
 ### 1. Upgrade Longhorn manager
 
 - To upgrade from v1.0.x, see [this section.](./longhorn-manager)
 
-### 2. Upgrade Longhorn Engine
+### 2. Manually Upgrade Longhorn Engine
 
 After Longhorn Manager is upgraded, Longhorn Engine also needs to be upgraded [using the Longhorn UI.](./upgrade-engine)
 
+# Note:
+Since Longhorn v1.1.1, we provide an option to help you automatically upgrade all Longhorn volumes (except for the [disaster recovery volumes](../../snapshots-and-backups/setup-disaster-recovery-volumes/)) to the new default engine version after upgrading Longhorn manager.
+This feature reduces the amount of manual work you have to do when upgrading Longhorn.
+To enable this feature, you just need to turn on the setting [Automatically upgrade volumes' engine to the default engine image after upgrading Longhorn](../../references/settings/#automatically-upgrade-volumes-engine-to-the-default-engine-image-after-upgrading-longhorn) in Longhorn UI.
+
+Longhorn doesn't automatically upgrade disaster recovery volumes to the new default engine image because it would trigger a full restoration for the disaster recovery volumes.
```

Comment:
Need to mention the concurrent limit of the automated engine upgrade.

Reply:
Thank you. I fixed it